src/main/java/Command.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Represents a command that can be executed.
+ */
+abstract class Command {
+    /**
+     * Execute the command that user calls.
```

Comment:
Maybe using Executes instead of Execute will be better in following the documentation practices.